shapefile/src/test/scala/geotrellis/shapefile/ShapeFileReaderSpec.scala

```diff
@@ -31,5 +34,19 @@ class ShapeFileReaderSpec extends AnyFunSpec with Matchers {
         data.keys.toSeq should be (Seq("LowIncome", "gbcode", "ename", "WorkingAge", "TotalPop", "Employment"))
       }
     }
+
+    // https://github.com/locationtech/geotrellis/issues/3445
+    it("should read UTF-8 MultiPolygons feature attributes") {
+      val path = "shapefile/data/shapefiles/demographics-utf8/demographics.shp"
+      val features = ShapeFileReader.readMultiPolygonFeatures(path, Charset.forName("UTF-8"))
+      features.size should be (160)
+
+      features.take(4).map(_.data("ename").asInstanceOf[String]) shouldBe Seq("南关街道", "七里烟香", "谢庄镇", "Cheng Guan Zhen")
+
+      val featuresInvalid = ShapeFileReader.readMultiPolygonFeatures(path)
+      val enames = featuresInvalid.take(4).map(_.data("ename").asInstanceOf[String])
+      // println(s""" ${enames.map(x => s"«${x}»").mkString(" <=> ")} """)
```

Comment:
oh, is it a test code and should be removed?

Reply:
You mean line 48, right? Nathan also brought it up. I forgot to remove it...sorry.